tests/integration/test_gitlab.py

```diff
@@ -24,8 +24,10 @@ def setUp(self):
         persistent_data_file = os.path.join(
             PERSISTENT_DATA_PREFIX, f"test_gitlab_data_{test_name}.yaml"
         )
-        PersistentObjectStorage().storage_file = persistent_data_file
-        if PersistentObjectStorage().mode == StorageMode.write and not self.token:
+        self.pos = PersistentObjectStorage()
```

Comment:
It does not make sense since it's a singleton. (We need to access that from anywhere.)